src/locales/ptBR.ts

```diff
@@ -142,6 +142,8 @@
     'preferences.editorIndentSize': 'Tamanho do recup',
     'preferences.editorKeymap': 'Funções das teclas',
     'preferences.editorPreview': 'Pré visualização',
+    'preferences.enableAutoFetchWebPageTitle':
+      'Trazer o título da página da Web ao colar a URL no editor',
```

Comment:
```suggestion
      'Trazer o título da página da Web ao colar uma URL no editor',
```

Reply:
Thank you for pointing out the mistranslation! I'll fix it.